cg/services/analysis_starter/configurator/file_creators/nextflow/sample_sheet/rnafusion.py

```diff
@@ -0,0 +1,32 @@
+from cg.apps.housekeeper.hk import HousekeeperAPI
+from cg.constants.constants import Strandedness
+from cg.models.rnafusion.rnafusion import RnafusionSampleSheetEntry
+from cg.services.analysis_starter.configurator.file_creators.nextflow.sample_sheet.creator import (
+    NextflowSampleSheetCreator,
+)
+from cg.store.models import Case, Sample
+from cg.store.store import Store
+
+
+class RNAFusionSampleSheetCreator(NextflowSampleSheetCreator):
```

Comment:
what does PEP say about abbreviations in class en names? Either way we should pick one way and go with it. example of conflicting naming: `RnafusionSampleSheetEntry`